src/tasks/CommissionsTask.py

```
@@ -491,9 +491,11 @@ def find_next_hint(self, x1, y1, x2, y2, s, box_name="hint_text"):
             return True
 
     def reset_and_transport(self):
+        # 1) 打开局内菜单（ESC 菜单）
         self.open_in_mission_menu()
         self.wait_until(
             condition=lambda: not self.find_esc_menu(),
+            # 2) 点击“设置”入口（局内菜单右下区域），点击后应关闭 ESC 菜单
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)